app/assets/stylesheets/application.scss

```diff
@@ -18,12 +18,16 @@
 
 @import 'bootstrap';
 @import 'bootstrap-toggle';
-@import 'font-awesome-sprockets';
-@import 'font-awesome';
+@import 'companies';
+@import 'custom';
+@import 'dashboard';
 @import 'maps';
 @import 'pagination';
 @import 'style';
-@import 'custom';
-@import 'dashboard';
+@import 'users';
+
 @import 'cookies_eu';
 @import 'shf_custom_cookies_eu';
+
+@import 'font-awesome-sprockets';
```

Comment:
Minor:  I would group the @import statements for all _gems_ together and put them before any of ours so that we can override any definitions, etc. in them.